aboutus.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            .center {
+                margin: auto;
+                width: 85%;
+                padding: 8px;
+            }
+            .pg {
+                line-height: 1.4;
+                font-size: 18px;
+            }
+            .float-container {
+                width: 90%;
+                margin: auto;
+                display: flex;
+                flex-wrap: wrap;
+                align-items: flex-start;
+
+            }
+            .float-child {
+                margin-left: 2.5%;
+                margin-right: 2.5%;
+                margin-bottom: -170px;
+                margin-top: 2.5%;
+                width: 44%;
+                float: left;
+                border-radius: 8px;
+                border-color: azure;
+                border-style: solid;                
+            }
+            a:link, a:visited {
+                color: aqua;
+            }
+
+            a:hover, a:active {
+                color:red;
+            }
+            .gallery{
+                border-radius: 8px;
+                border-color: azure;
+                border-style: solid;
+                width: 90%;
+                margin: auto;
+                display: flex;
+                flex-wrap: wrap;
+                align-items: flex-start;
+            }
+            .image{
+                width: fit-content;
+                height: fit-content;
+                float: left;
+            }
+
+        </style>
+    </head>
+    <Body style="background-color: #323232; color: azure; font-family: Arial, Helvetica, sans-serif; padding: 15x; padding-top: 30px">
+        <title>
+            About our team
+        </title>
+        <h1 align="center">
+            About our team
+        </h1>
+        <div class="center">
+            <p class="pg"> 
+                Our team is a group of passionate FTC and FRC students and alumni who want to promote FIRST team's choice of what devices they use. Our team includes:
+                <br>
+                <br>Iris, an FTC15031 member. <a href="https://github.com/Iris-TheRainbow ">Her github is here.</a>
```

Comment:
Make the actual "name, a team member" text the hyperlink, instead of in a second sentence?
Also, "Name from team TEAMNUM" or "Name, mentor/mentor of team TEAMNUM" might be better?